inc/functions/options.php

```diff
@@ -48,9 +48,15 @@ function update_rocket_option( $key, $value ) { // phpcs:ignore WordPress.Naming
  * @source wp-admin/includes/plugin.php
  *
  * @param string $plugin Plugin folder/main file.
+ *
+ * @return boolean true when plugin is active; else false.
  */
 function rocket_is_plugin_active( $plugin ) {
-	return in_array( $plugin, (array) get_option( 'active_plugins', [] ), true ) || rocket_is_plugin_active_for_network( $plugin );
+	return (
+		in_array( $plugin, (array) get_option( 'active_plugins', [] ), true )
+		||
+		rocket_is_plugin_active_for_network( $plugin )
+	);
```

Comment:
are the wrapping parenthesis required?

Reply:
Same answer as above. We are returning the result of a conditional expression. 

They are unnecessary. I use the `( )` to show "conditional expression" intent and a boolean will be returned.

Reply:
Let's discuss our preferred standard for returning the result of a conditional expression. This code is a great example.

This code is a conditional expression. It returns the result of `in_array()` OR `rocket_is_plugin_active_for_network()`.

```php
return (
		in_array( $plugin, (array) get_option( 'active_plugins', [] ), true )
		||
		rocket_is_plugin_active_for_network( $plugin )
	);
```

For me, I find this formatting to be very clear and understandable.  Why?

- By wrapping it in `( )`, it tells us it's a conditional expression. 
- It looks just like an `if` conditional, i.e. making it easily recognizable and understandable.
- With the conditional operator on its own line, we can quickly see that it's an OR expression.

@Tabrisrp and @crystinutzaa What do you think? Do you like or not like this formatting and design pattern as part of our coding standards? Let's discuss.